packages/myTestCopy/test/myTestCopy.ts

```diff
@@ -0,0 +1,142 @@
+// limitations under the License.
+
+import myProbotApp from '../src/myTestCopy';
+
+import { expect } from 'chai';
+import { resolve } from 'path';
+import { Probot } from 'probot';
+import snapshot from 'snap-shot-it';
+
+import nock from 'nock';
+nock.disableNetConnect();
+
+const fixturesPath = resolve(__dirname, '../../test/fixtures');
+
+// TODO: stop disabling warn once the following upstream patch is landed:
+// https://github.com/probot/probot/pull/926
+global.console.warn = () => {};
+
+describe('ConventionalCommitLint', () => {
+  let probot: Probot;
+
+  beforeEach(() => {
+    probot = new Probot({
+      // use a bare instance of octokit, the default version
+      // enables retries which makes testing difficult.
+      Octokit: require('@octokit/rest'),
+    });
+
+    const app = probot.load(myProbotApp);
+    app.app = {
+      getSignedJsonWebToken() {
+        return 'abc123';
+      },
+      getInstallationAccessToken(): Promise<string> {
+        return Promise.resolve('abc123');
+      },
+    };
+  });
+
+  it('sets a "failure" context on PR, if commits fail linting', async () => {
+    const payload = require(resolve(
+      fixturesPath,
+      './pull_request_synchronize'
+    ));
+    const invalidCommits = require(resolve(fixturesPath, './invalid_commit'));
+    const requests = nock('https://api.github.com')
+      .get('/repos/bcoe/test-release-please/pulls/11/commits?per_page=100')
+      .reply(200, invalidCommits)
+      .post('/repos/bcoe/test-release-please/check-runs', body => {
+        snapshot(body);
+        return true;
+      })
+      .reply(200);
+
+    await probot.receive({ name: 'pull_request', payload, id: 'abc123' });
+    requests.done();
+  });
+
+  it('sets a "success" context on PR, if commit lint succeeds', async () => {
+    const payload = require(resolve(
+      fixturesPath,
+      './pull_request_synchronize'
+    ));
+    const validCommits = require(resolve(fixturesPath, './valid_commit'));
+
+    const requests = nock('https://api.github.com')
+      .get('/repos/bcoe/test-release-please/pulls/11/commits?per_page=100')
+      .reply(200, validCommits)
+      .post('/repos/bcoe/test-release-please/check-runs', body => {
+        snapshot(body);
+        return true;
+      })
+      .reply(200);
+
+    await probot.receive({ name: 'pull_request', payload, id: 'abc123' });
+    requests.done();
+  });
+
+  describe('PR With Multiple Commits', () => {
+    it('has a valid pull request title', async () => {
+      const payload = require(resolve(
+        fixturesPath,
+        './pull_request_synchronize'
+      ));
+      // create a history that has one valid commit, and one invalid commit:
+      const invalidCommits = require(resolve(fixturesPath, './invalid_commit'));
+      invalidCommits.push.apply(
+        invalidCommits,
+        require(resolve(fixturesPath, './valid_commit'))
+      );
+
+      const requests = nock('https://api.github.com')
```

Comment:
we can probably trim down this test quite a bit to only enforce the branch protection checks.

I would just pull the start of the commit checks onto a branch, so that you still have them as a starting point for future work.

Reply:
ah, I didn't mean to commit myTestCopy to this branch - will remove. Thanks!